Rock.Blocks/Core/PersonFollowingList.cs

```diff
@@ -127,17 +127,20 @@ private Person GetSpouse( Person person )
             var adultGuid = Rock.SystemGuid.GroupRole.GROUPROLE_FAMILY_MEMBER_ADULT.AsGuid();
             var marriedGuid = Rock.SystemGuid.DefinedValue.PERSON_MARITAL_STATUS_MARRIED.AsGuid();
 
-            return person.Members.Where( m =>
-                    person.MaritalStatusValue != null && 
-                    person.MaritalStatusValue.Guid.Equals( marriedGuid ) &&
-                    m.GroupRole.Guid.Equals( adultGuid ) )
+        // Early null check to return null if not married or no spouse found
```

Comment:
There's still an indentation issue here (either a tab or spacing problem), but I'm going to merge it and then adjust as needed.  Try looking into that for next time.